test/httpoison_base_test.exs

```diff
@@ -569,6 +569,32 @@ defmodule HTTPoisonBaseTest do
              }
   end
 
+  test "request returns MaybeRedirect when passing follow_redirect option" do
```

Comment:
So I'm not sure what I'm doing wrong but I can't actually get `mix test` to run. Looking at the error it seems like `httparrot` requires some config to be set but there isn't any config for this app. 

How best would I go about testing this?

Reply:
I will check it out!

Reply:
Running the test the only errors I got were related to http versus https 🤔 If you change everything to `http://` it seems to work fine.

Reply:
Alright cool, I'll make that change then thanks 🙂

Reply:
I found the error; for the sake of asserting that the `redirect_url` does actually read from the response of the redirect headers I kept it as `https` vs `http` but fixed my mistake 😅